packages/forms/src/deprecated/Form.js

```diff
@@ -122,16 +120,9 @@ class Form extends React.Component {
 
 	render() {
 		if (this.props.uiform) {
-			const props = Object.assign({}, this.props);
-			props.moz = true;
-			if (props.widgets) {
-				Object.keys(props.widgets)
-					.filter(key => props.widgets[key].displayName !== 'TFMigratedWidget')
-					.forEach(key => {
-						props.widgets[key] = wrapCustomWidget(props.widgets[key]);
-					});
-			}
-			return <UIForm {...props} />;
+			return (
+				<div className="alert alert-danger">The combination of old UISpec with props.uiform is not supported anymore.</div>
```

Comment:
You removed, the compat mode, this need to be documented in a migration guide.
Perhaps you can add it in the `breaking changes` part of this PR description, so we can validate it and we just have to copy/paste in the real migration guide

Reply:
good point, done